components/cache/cache_items.rst

```diff
@@ -0,0 +1,116 @@
+.. index::
+   single: Cache Item
+   single: Cache Expiration
+   single: Cache Exceptions
+
+Cache Items
+===========
+
+Cache items are each one of the information units stored in the cache as a
+key/value pair. In the Cache component they are represented by the
+:class:`Symfony\\Component\\Cache\\CacheItem` class.
+
+Cache Item Keys and Values
+--------------------------
+
+The **key** of a cache item is a UTF-8 encoded string which acts as its
+identifier, so it must be unique for each cache pool. The PSR-6 standard limits
+the key length to 64 characters, but Symfony allows to use longer keys (they are
+encoded internally to reduce their size).
```

Comment:
It does not matter what Symfony Cache does internally. The interface only guarantees that keys up to 64 characters will work. Say, the user swap out Symfony Cache for another PSR-6 cache, things might break if they go over that limit. And we should encourage users to stay within the limit - warn them of the risk of doing otherwise.